src/components/input/input.js

```diff
@@ -569,7 +569,8 @@ function inputTextareaDirective($mdUtil, $window, $mdAria, $timeout, $mdGesture)
 
         function onDrag(ev) {
           if (!isDragging) return;
-          element.css('height', startHeight + (ev.pointer.y - dragStart) - $mdUtil.scrollTop() + 'px');
+
+          element.css('height', (startHeight + ev.pointer.distanceY) + 'px');
```

Comment:
It's under the hood the same. Kristyian just calculated the distance manually, but `$mdGestures` already does this under the hood (common usage)